scripts/gh-projects/examples/mux-video-integration/phase-2/c3.md

```
@@ -0,0 +1,14 @@
+Wire Mux Data analytics into the MuxPlayer element.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)